repoman/lib/repoman/scanner.py

```diff
@@ -93,6 +93,10 @@ def __init__(self, repo_settings, myreporoot, config_root, options,
 			'profile_list': profile_list,
 			'pmaskdict': global_pmaskdict,
 			'lic_deprecated': liclist_deprecated,
+			'package.deprecated': set(atom.cp for atom in
```

Comment:
It might be useful to support entries like `x11-libs/gtk+:2` or `=x11-libs/gtk+-2*`...
(Commenting here because this `.cp` would be unnecessary.)

Reply:
That would complicate the logic in _depend_checks.py since we would have to match dependencies to dependencies (whereas dependencies normally match to packages).

Reply:
Fixed like this:
```python
all_deprecated = False
for pkg_match in portdb.xmatch("match-all", atom):
	if any(repo_metadata['package.deprecated'].iterAtomsForPackage(pkg_match)):
		all_deprecated = True
	else:
		all_deprecated = False
		break
```